packages/documentation/src/stories/getting-started/components-angular/angular-app-component.sample.html

```diff
@@ -0,0 +1,3 @@
+<!-- app.component.html -->
+
+<post-icon *ngIf="condition" [attr.name]="iconName"></post-icon>
```

Comment:
Is `attr.name` correct? I thought that it will be converted to an Angular prop.

Reply:
In my testing both seems to work. Is there a preferred way? Do we want to document it somewhere?